Pipfile.lock

```diff
@@ -985,6 +985,40 @@
             ],
             "version": "==2.8.3"
         },
+        "psycopg2-binary": {
```

Comment:
This should have recently been removed as a dependency. The binary is nice, but other things already require non binary psycopg2, so I removed it from the Pipfile.

Reply:
Okay. I'll pull it back out.

I did a `pipenv clean` recently and saw it get removed, but thought it was an error. Glad to know it's not. :smiley: